index.js

```diff
@@ -40,7 +49,83 @@ function toggleColor(element) {
   }
 }
 
+const header = document.querySelector("h1#header")
+
+header.addEventListener("click", function() {
+  toggleColor(document.querySelector("h1#header"))
+})
+
+// function toggleRedBlack(event) {
+//   if (event.target.style.color === "red") {
+//     event.target.style.color = "black"
+//   } else {
+//     event.target.style.color = "red"
+//   }
+// }
+
+// const header = document.querySelector("h1#header")
+
+// header.addEventListener("click", toggleRedBlack)
 
 /***** Deliverable 2 *****/
 
+const newPlayerForm = document.querySelector("#new-player-form")
+
+newPlayerForm.addEventListener("submit", function(event) {
+  event.preventDefault()
+
+  const number = event.target.number.value
+  const name = event.target.name.value
+  const nickname = event.target.nickname.value
+  const photo = event.target.photo.value
+
+  const PlayerObj = {
+    number: number,
+    name: name,
+    nickname: nickname,
+    photo: photo,
+    likes: 0
+  }
+
+  renderPlayer(PlayerObj)
```

Comment:
For the `PlayerObj` - try to stick with lower case names for variables (in JS, upper case variable names are usually used for classes, which we'll see later on). Also, if you want to tighten this code up a bit: 

```js
  const playerObj = {
    number: event.target.number.value,
    name: event.target.name.value,
    nickname: event.target.nickname.value,
    photo: event.target.photo.value,
    likes: 0
  }
```